src/app/lib/analyticsUtils/analytics.const.ts

```diff
@@ -0,0 +1,18 @@
+export const SUPPORTED_MEDIUM_CAMPAIGN_TYPES = [
+  'affiliate',
+  'sl',
+  'email',
+  'display',
+  'custom',
+  'RSS',
+];
+
+export const MEDIUM_CAMPAIGN_IDENTIFIER = 'at_medium';
+
+export const XTOR_CAMPAIGN_IDENTIFIER = 'xtor';
+
+export const VIEW_EVENT = 'view';
+export const CLICK_EVENT = 'click';
+
+export const LITE_ATI_VIEW_TRACKING = 'data-lite-ati-view';
```

Comment:
Might be drastic, but if this constant changes, it's going to break the [lite view tracking code](https://github.com/bbc/simorgh/blob/94345860c682ea02114ff4cc0e516a9e14f8a18b/src/server/utilities/liteATITracking/viewTracking/index.ts#L12), do we need to have some form of snapshot test to warn us?

Reply:
Yeah maybe we should do that, just in case. We did try importing the value and had problems, so I'll add a cheeky unit test somewhere to make sure that the value in the lite script matches this. I'd like to think that it would be caught elsewhere, but no harm in doubling down 😸

Reply:
Yeah importing anything in scripts that will be inlined won't work because webpack doesn't inline imported modules https://github.com/bbc/simorgh/pull/12592#issuecomment-2792262353

Reply:
I've added a test for it: https://github.com/bbc/simorgh/blob/1e4d865d44de5c0f4562b45d8b832a41829c6805/src/server/utilities/liteATITracking/viewTracking/index.test.ts#L15-L27